cloud/openstack/os_server.py

```diff
@@ -75,7 +75,7 @@
      default: None
    security_groups:
      description:
-        - The name of the security group to which the instance should be added
+        - A list of security groups to which the instance should be added
      required: false
      default: None
```

Comment:
Change this to document what the code actually has (the security_group called "default")?